page.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <script src="https://kit.fontawesome.com/138cf4be1d.js" crossorigin="anonymous"></script>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <title>Akpuru Solomon Website</title>
+</head>
+
+<body>
+  <div id="container">
+   <header>
+      <div class="topnav" id="myTopnav">
+        <a class="logo" href="#">Akpuru Solomon</a>
+        <a href="#works" class="active">Portfolio</a>
+        <a href="#about">About</a>
+        <a href="#contact">Contact</a>
```

Comment:
- [ ] Please make sure everything in the Figma design is implemented in your project. There is an envelop icon on the topbar of the Figma design but you didn't add this icon. Please add this icon.  
- [ ] Please change the color of the text to white so that it looks like the design. 

<p align="center">&nbsp;
 
| Figma design |  Your project |
| ---------- | ----------- |
 | ![Template 2-Main](https://user-images.githubusercontent.com/12524453/178104127-595026a9-0a1b-4335-8648-bea243ef3d97.png) | ![Template 2-project](https://user-images.githubusercontent.com/12524453/178104242-c2e1feff-b67e-4368-90bb-065c7257733a.png) |
</p>